test/e2e_node/topology_manager_test.go

```diff
@@ -28,6 +27,8 @@ import (
 	"sync"
 	"time"
 
+	"k8s.io/utils/cpuset"
+
```

Comment:
nit: I Think this would be better without this space

Reply:
fixed, thanks!